pkg/cmd/cli/describe/chaindescriber.go

```diff
@@ -78,7 +78,7 @@ func (d *ChainDescriber) MakeGraph() (osgraph.Graph, error) {
 // image stream tag (name:tag) in namespace. Namespace is needed here
 // because image stream tags with the same name can be found across
 // different namespaces.
-func (d *ChainDescriber) Describe(ist *imageapi.ImageStreamTag) (string, error) {
+func (d *ChainDescriber) Describe(ist *imageapi.ImageStreamTag, buildInputEdgeKinds []string) (string, error) {
```

Comment:
yup, a boolean flag would work (we only ever want triggers or triggers +
input)

On Fri, Jul 31, 2015 at 2:08 PM, David Eads <notifications@github.com>
wrote:

> In pkg/cmd/cli/describe/chaindescriber.go
> <https://github.com/openshift/origin/pull/3748#discussion_r35999620>:
>
> > @@ -78,7 +78,7 @@ func (d *ChainDescriber) MakeGraph() (osgraph.Graph, error) {
> >  // image stream tag (name:tag) in namespace. Namespace is needed here
> >  // because image stream tags with the same name can be found across
> >  // different namespaces.
> > -func (d *ChainDescriber) Describe(ist *imageapi.ImageStreamTag) (string, error) {
> > +func (d *ChainDescriber) Describe(ist *imageapi.ImageStreamTag, buildInputEdgeKinds []string) (string, error) {
>
> How would you feel about passing a flag or having different describe
> methods or something to indicate intent instead of leaking knowledge of the
> flag library to callers?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3748/files#r35999620>.
>